frontend/app/routes/create.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
+
+export default class TeachCreateRoute extends Route {}
```

Comment:
should the content create be authenticated first? If no, then the auth check should be removed on the backend endpoint